packages/picasso.js/src/web/components/tooltip/render.js

```diff
@@ -0,0 +1,62 @@
+import extend from 'extend';
+
+function resolveClasses(props, opts) {
+  return {
+    tooltip: typeof props.tooltipClass === 'function' ? props.tooltipClass({ dock: opts.dock }) : props.tooltipClass,
```

Comment:
So, tooltipClass, contentClass and arrowClass can be a function as well? Shouldn't that be specified in the API reference?

Reply:
Most, if not all, settings can be a function, but to keep the API clean. I prefer to specify only the required type and instead use the documentation to add examples where you use a function. That also adds some flexibility when it comes to breaking changes.